src/Confluent.Kafka/SerializationContext.cs

```diff
@@ -37,10 +37,16 @@ public static SerializationContext Empty
         /// <param name="topic">
         ///     The topic the data is being written to or read from.
         /// </param>
-        public SerializationContext(MessageComponentType component, string topic)
+        /// <param name="headers">
+        ///     The collection of message headers (or null). Specifying null or an 
+        ///      empty list are equivalent. The order of headers is maintained, and
+        ///     duplicate header keys are allowed.
+        /// </param>       
+        public SerializationContext(MessageComponentType component, string topic, Headers headers)
```

Comment:
add a default value for headers of `null`, that way it's backwards compatible (after re-compile).

Reply:
And done :)